### PR TITLE
Doc/taipy#1704 - Timestamp properties for Job and Submission

### PR DESCRIPTION
**Pull request by trgiangdo**

### CODE REVIEW

File: docs/userman/scenario_features/sdm/submission/index.md [hunk 1]
```diff
@@ -28,12 +28,12 @@ For more details and examples on how to submit scenarios, sequences or tasks, se
 Here is the list of the `Submission^`'s attributes:
```
Comment (jrobinAV) [inline docs/userman/scenario_features/sdm/submission/index.md]:
```suggestion
Here is the list of the main `Submission^`'s attributes:
```

File: docs/userman/scenario_features/sdm/job/index.md [hunk 2]
```diff
@@ -50,11 +50,11 @@ A `Job^` entity ìs identified by a unique identifier `id` that is generated by
 
 Here is the list of the `Job^`'s attributes:
```
Comment (jrobinAV) [inline docs/userman/scenario_features/sdm/job/index.md]:
```suggestion
Here is the list of the main `Job^`'s attributes:
```

**Review by jrobinAV — APPROVED** [review]
Neat

File: docs/userman/scenario_features/sdm/submission/index.md (hunk 1 above)
Comment (trgiangdo) [reply docs/userman/scenario_features/sdm/submission/index.md]: How about

```suggestion
Here is the list of `Submission^`'s main attributes:
```

File: docs/userman/scenario_features/sdm/job/index.md (hunk 2 above)
Comment (trgiangdo) [reply docs/userman/scenario_features/sdm/job/index.md]: How about

```suggestion
Here is the list of `Job^`'s main attributes:
```

File: docs/userman/scenario_features/sdm/submission/index.md (hunk 1 above)
Comment (jrobinAV) [reply docs/userman/scenario_features/sdm/submission/index.md]: much better !

**Pull request merged**